README.md

```diff
@@ -2,221 +2,62 @@ Category
 ========
 
 * [Hardware Versions](https://github.com/96boards-hikey/tools-images-hikey960/blob/master/README.md#hardware-versions)
-* [Rescue using hisi_idt](https://github.com/96boards-hikey/tools-images-hikey960/blob/master/README.md#rescue-using-hisi_idt)
-* [Rescue From Endless-Rebooting](https://github.com/96boards-hikey/tools-images-hikey960/blob/master/README.md#rescue-from-endless-rebooting) 
-* [Flash Images by Bootloader](https://github.com/96boards-hikey/tools-images-hikey960/blob/master/README.md#flash-images-by-bootloader)
-* [How to create boot.img and dt.img](https://github.com/96boards-hikey/tools-images-hikey960/blob/master/README.md#how-to-create-bootimg-and-dtimg)
+* [Simple Rescue Flashing](https://github.com/96boards-hikey/tools-images-hikey960/blob/master/README.md#simple-rescue-flashing)
 
 Hardware Versions
 =================
 
 There are two versions of HiKey960: v1 and v2. One important difference you should know is: v1 is using Jumper (J2001), while v2 is using DIP switch (SW2201). Both are located in the bottom side of board.
 
```

Comment:
Not sure what else to refer them to. Having it at least match the schematic is good in my mind. So probably going to leave it here, as there aren't other jumpers/switches on the bottom of the board.